coap-gateway/bootstrapper/go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/plgd-dev/cloud/coap-gateway/bootstrapper
```

Comment:
pls remove `go.mod` file - we use only per repository so `github.com/plgd-dev/cloud/go.mod`

Reply:
done